### PR TITLE
Day 7 reg exp data to a file

### PR DESCRIPTION
**Pull request by Dino-Herbert**
I deleted the directories with everyone else's names from 001 on my local machine.  Hopefully this doesn't cause a problem with my commit.

### CODE REVIEW

File: 2_adventures/001/Dino-Herbert/07_Apr19/reg_exp_test3.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  file2.write(b)
+      
+end
+
+
+ 
+ 
+ 
+
+
```
Comment (kotp) [inline 2_adventures/001/Dino-Herbert/07_Apr19/reg_exp_test3.rb:28]: Yes, @patmaddox grinding my teeth.  Not a big deal, but the extra lines are not necessary. :) 23 to 28 can be gone, so could 20, and 22.  Though 22 is more of a personal choice for sure.

**Comment by kotp** [comment]
lgtm

**Comment by Dino-Herbert** [comment]
The text file has a bunch of square brackets in place of the other data in the file.  Can anyone shed some light on why this is?  Any suggestions to create the file with only the information I want?  When I "puts" to the terminal, it returned exactly what I wanted.  Thanks!

**Comment by kotp** [comment]
If you can generate exactly what you want with puts, perhaps you can run your program and pipe that into the file?  Not sure exactly what the goal is.  Maybe I will fork your code and see if I can make a pull request.

**Comment by Dino-Herbert** [comment]
@kotp Your suggestion helped my to realize that I can puts to the screen as well as to a file.  This did the trick.  Thanks a lot.

**Comment by kotp** [comment]
@Dino-Herbert I sent a pull request anyway... deny it or approve it... no matter.

**Comment by patmaddox** [comment]
Go you!

**Comment by patmaddox** [comment]
@Dino-Herbert this doesn't seem to have deleted anyone's files... so it's all good!

**Pull request merged**